src/coreclr/jit/codegencommon.cpp

```diff
@@ -11409,7 +11408,6 @@ void CodeGen::genRegCopy(GenTree* treeNode)
         //
         // There should never be any circular dependencies, and we will check that here.
 
-        GenTreeCopyOrReload* copyNode = treeNode->AsCopyOrReload();
```

Comment:
If some of these locals exist to aid debugging, I am happy to revert. Just let me know...